src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -1,6 +1,6 @@
 @import 'src/styles/common';
 
-$container_max_width: 1860px;
+$container_max_width: 1770px;
```

Comment:
Not sure if for this PR or for another one; but could we do it as a `calc` of maximum BLAST content width (or whatever should be the name of the 1800px that we use for the BLAST navigation in the top bar) minus the standard gutter that we have on the left? Otherwise, 1770px is a bit magical.

Reply:
Its a minor change, doesnt require another PR.